Mvc.NoFun.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio 14
-VisualStudioVersion = 14.0.22522.0
+VisualStudioVersion = 14.0.22526.0
```

Comment:
this is the churn of different people having different vs versions. I guess do whatever here